src/sass/style.scss

```diff
@@ -19,12 +15,9 @@ $image-path: "~uswds/src/img";
 @import "modules/m-vet-nav";
 @import "~@department-of-veterans-affairs/jean-pants/sass/modules/m-print";
 @import "~@department-of-veterans-affairs/jean-pants/sass/modules/m-dropdown";
-@import "modules/m-nav-sidebar";
 @import "modules/m-overlay";
-@import "modules/m-education-wizard";
-@import "modules/m-applications";
 @import "~@department-of-veterans-affairs/jean-pants/sass/modules/m-additional-info";
-@import "modules/m-downtime-notification";
+@import "../platform/monitoring/DowntimeNotification/sass/m-downtime-notification";
```

Comment:
Maybe this should be imported by React app styles? Not sure, so I left it alone.

Reply:
^ Good call, I can add that into the DowntimeNotification directory. Should I do that in this branch or hold off?

Reply:
It's in the right directory now, but I was thinking that we could just import this in the sass files for each React app that uses it. But it's only a few lines, so I don't think it matters that much.